recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  path: .
+
+requirements:
+  build:
+    - python
+    - param
+    - numpy
```

Comment:
You're right it shouldn't be needed for the build step. I know I added it for some reason at some point but it is likely I simply got myself confused. I'll try removing it now as there is no such compilation step required (HoloViews is pure Python).